runtime-tests/runtimeTests.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { endpoints } from './endpoints';
+import { PORT, URL_PATH } from './helpers/consts';
+import { request } from './helpers/request';
+import { BlockResponse, ChainConfig, ChainSpec } from './types';
+
+const config = JSON.parse(process.env.__CONFIGURATION as string) as ChainConfig;
+const chain = config.chain as ChainSpec;
+
+const polkadotEndpoints: string[][] = endpoints[chain];
+
+describe('Runtime Tests for blocks', () => {
+	jest.setTimeout(15000);
+
+	/**
+	 * Test runtimes for `/blocks`
+	 */
+	test.each(polkadotEndpoints)(
+		'Given path %p, it should return block height %p',
+		async (blockPath, blockHeight) => {
+			const res = await request(blockPath, URL_PATH, PORT);
+			const responseJson = JSON.parse(res) as BlockResponse;
+
+			expect(responseJson['number']).toBe(blockHeight);
```

Comment:
In the future we should just take a snapshot of the response, save them and then compare the responses those save snapshots. That way we would essentially have an e2e user api contract testing

Reply:
I very much like this idea, do you think I should just add it in this PR?

Reply:
Either sounds good to me! Will be a super valuable feature to be able to show how responses change with their diffs

Reply:
2 part PR it is! :)